modules/standard/IO.chpl

```diff
@@ -22,9 +22,6 @@
 
 Support for a variety of kinds of input and output.
 
-.. note:: All Chapel programs automatically ``use`` this module by default.
-          An explicit ``use`` statement is not necessary.
-
```

Comment:
I think instead this should list and link to the functions that are available by default

Reply:
How does the new note look?

Reply:
Maybe
```
.. note:: All Chapel programs automatically include :proc:`~ChapelIO.write`,
          :proc:`~ChapelIO.writeln` and :proc:`~ChapelIO.writef`. Please ``use IO`` to access other features in this module.
```

Reply:
We already automatically generate a sample use statement for standard modules.  IO was one of the few ones where we said "but actually, you don't need that use statement"

Reply:
It's not a burden to add it, but that's why I didn't do it on my own, so I'd like to check if that additional information is sufficient.  (This is how it looks on 1.20 today: https://chapel-lang.org/docs/latest/modules/standard/IO.html)

Reply:
OK, then what you have looks good to me.